hello-rails-react/src/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import MessagesPage from './pages/Messages';
+
+class App extends React.PureComponent {
+  render() {
+    return (
+      <Router>
+        <Routes>
+          <Route path="/" element={<MessagesPage />} />
+        </Routes>
```

Comment:
* After adding some messages in the database and running the rails server I couldn't display random greetings below are screenshots of my errors 
![image](https://user-images.githubusercontent.com/52497246/196312562-3045ff95-ef05-4841-a71d-104867688b52.png)

- But as you can see in this second screenshot that the API URL indeed does return random greetings but the react section of the app isn't working as expected, Kindly connect the react part properly and ensure when the rails server starts that the view renders random messages 
![image](https://user-images.githubusercontent.com/52497246/196313469-69de65af-70ad-43be-873e-320d60275177.png)

Reply:
Hello @Davidosky007, please run the React server `npm start` inside the hello-rails-react folder. I made them run into two separate servers inside one app.